contracts/token/psp1155/macro/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "psp1155_macro"
```

Comment:
pls rename to psp1155_derive - that's the standard naming of derive macro crates